Sandy Documentation Overhaul (#3799)
freesewing/freesewing: markdown/org/docs/patterns/sandy/instructions/en.md

</Note>

## Step 1: Prepping the Skirt

- If including a closure, sew the skirt seam  _good sides together_ up to where you intend the opening to start.
- Add Pockets if using.
- If using lining, prep the same as the skirt.
- _Finish_ seams if not lining.

<Note>

Pockets are not included in Sandy as it has one seam by default or no seams at all.

If you would like inseam pockets you can cut the skirt pattern piece into multiple pieces rather than a single one to create seams to insert them into. Cutting it into thirds (for default) or two (for seamless) will give you two sideseams for two inseam pockets, just don't forget to add back seam allowance to the cut lines if including.

If you do not wish to create additional seams but still wish to have pockets, you can use Patch Pockets or if you are feeling adventurous, you can use Welt Pockets.

</Note>

## Step 2: Prep the opening

- Insert zipper or placket into opening if using.

> - If not using, press the openings seam allowance to the inside and _Edgestitch_/_Topstitch_ in place. You may also wish to continue the topstiching down the seam.

Reviewer: ```suggestion
- If not using, press the openings seam allowance to the inside and _Edgestitch_/_Topstitch_ in place. You may also wish to continue the topstitching down the seam.
```

<Note>

Skip this step if:
- You are including the zipper in the waistband.

> - You are making a seamless version

Reviewer: ```suggestion
- You are making a seamless version.
```

</Note>

## Step 3: Lining

> - (Optional) Interline the skirt with a stiffener.

Reviewer: ```suggestion
- (Optional) Interline the skirt with a stiffer or stronger fabric.
```

- Attach Lining to skirt at hem and opening by your preferred method.
- _Baste_ Lining to skirt at waist.
- Gather the waist if needed.

<Note>

## Step 4: The waistband

### Closures

- Interface/stiffen half the waistband part lengthwise. Or interface/stiffen one of the curved waistband parts.
- Press under the seam allowance on the long edge of the waistband that is not interfaced. Trim. _Baste_ the fold in place if needed.

> - With _good sides together_ sew the interfaced waistband/side of the waistband to the skirt. Gather the skrit down to the waistband if needed.

Reviewer: ```suggestion
- With _good sides together_ sew the interfaced waistband/side of the waistband to the skirt. Gather the skirt down to the waistband if needed.
```

> - If curved waistband sew the non-interfaced part to the interfaced part. Trim and clip seam and press the non-interfaced part up and away from the skirt.

Reviewer: ```suggestion
- If using the curved waistband sew the non-interfaced part to the interfaced part. Trim and clip seam and press the non-interfaced part up and away from the skirt.
```

There will be some overhangs:

- The side you wish not to overlap should be overhang by your seam allowance.
- The side intended to overlap will have a greater overhang.
- If using a placket, both sides will overhang by your seam allowance.
- If using a zipper in the waistband the greater overhang will need to be trimmed to the seam allowance.

__Zipper in Waistband__

> - If inserting a zipper into the waistband now is the time to do so, attach the zipper from the waistband fold line down. Or for from seam-line down for curved waistband.

Reviewer: ```suggestion
- If inserting a zipper into the waistband now is the time to do so, attach the zipper from the waistband fold line down. Or from seam-line down for curved waistband.
```

> - _Slipstich_ or _Whipstitch_ the lining to the zipper at this point if you have not treated the lining and skirt as one at the opening.
> - Press under overhangs.  
> - Press the waistband _wrong sides together_ along fold-line. Or along seam-line for curved waistband.
> - Slipstich_ or _Whipstitch_ the overhang edges to the zipper.
> - _Egdestitch_ the waistband in place.
> - Alternatively, _Slipstich_ or _Whipstitch_ the waistband in place on the inside.
> 
> __Other closures__ 
> - Press the waistband _good sides together_ along fold-line. Or along seam-line for curved waistband.

Reviewer: ```suggestion
- _Slipstitch_ or _Whipstitch_ the lining to the zipper at this point if you have not treated the lining and skirt as one at the opening.
- Press under overhangs.  
- Press the waistband _wrong sides together_ along fold-line. Or along seam-line for curved waistband.
- Slipstitch_ or _Whipstitch_ the overhang edges to the zipper.
- _Edgestitch_ the waistband in place.
- Alternatively, _Slipstitch_ or _Whipstitch_ the waistband in place on the inside.

__Other closures__ 
- Press the waistband _good sides together_ along fold-line. Or along seam-line for curved waistband.
```
